corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -185,7 +185,7 @@ hqDefine("cloudcare/js/formplayer/app", function () {
     });
 
     FormplayerFrontend.on('startForm', function (data) {
-        FormplayerUtils.stopIntervalSync();
+        FormplayerFrontend.permitIntervalSync = false;
```

Comment:
Does this change basically mean that the next time there _would_ be an interval sync, it won't happen because this is set to false?

Reply:
Yes. We don't want to sync while inside a form (based on what i read in this [PR](https://github.com/dimagi/formplayer/pull/751)) 

But I also want to keep the 5 minute interval so I can't stop and restart the interval .